metadata.json

```diff
@@ -24,7 +24,8 @@
         "6",
         "7",
         "8",
-        "9"
+        "9",
+        "9-arm"
```

Comment:
Is that the official format? Is there a difference between 9 vs 9-arm support?

Reply:
@bastelfreak we are hoping to extend testing to cover arm architectures, and I _believe_ this is what the format will be. This sample module is used to test our internal VM provisioning services.

Reply:
mhm so all modules would need to update their metadata.json? Is that something that should be discussed in https://github.com/puppetlabs/puppet-specifications/

Reply:
Similar discussion in https://github.com/puppetlabs/puppetlabs-stdlib/pull/1404#discussion_r1370097597

Reply:
@bastelfreak yeah was using that as my reference for moving forward with this. Have rethought the approach.

Reply:
Related discussion: https://github.com/puppetlabs/puppetlabs-stdlib/pull/1404#discussion_r1370097597

I think we need to have a good discussion about this. For example, does this mean other arches like s390x are unsupported? Why is this even a thing, because most of the code is noarch anyway.

Reply:
@ekohl I believe its been agreed that the `metadata.json` will stay as is and not specify the different architectures. I will defer to the folks in the modules team as they will be best placed to have this discussion.